src/components/RecipeView.vue

```diff
@@ -67,6 +67,12 @@
                         <strong>{{ t("cookbook", "Servings") }}: </strong
                         >{{ $store.state.recipe.recipeYield }}
                     </p>
+                    <p v-if="$store.state.recipe.aggregateRating != null">
+                        <strong>{{ t("cookbook", "Rating") }}: </strong
+                        >
+                        <span v-for="i in parseInt($store.state.recipe.aggregateRating.ratingValue)" class="icon-favorite rating-icon" />
+                        <span v-for="i in 5-parseInt($store.state.recipe.aggregateRating.ratingValue)" class="icon-favorite-white rating-icon" />
```

Comment:
You might want to use mdi components for the icons as demonstrated [here](https://nextcloud-vue-components.netlify.app/#/Components/NcActions).
Also you could add half-star icons for values near  `n.5`. E.g., `n.25` < `rating` < `n.75` has  `n + a half` star?

Reply:
Ah hadn't seen that before, I'll dive into that now. I'm going to have to resubmit this PR anyway as I've missed the sign off, and I now have 5 commits for as many lines of code I've added which is just unneeded.